src/output/file_name.rs

```diff
@@ -301,11 +315,16 @@ impl<'a, 'dir, C: Colours> FileName<'a, 'dir, C> {
 
         match self.file {
             f if f.is_directory()        => self.colours.directory(),
+            #[cfg(unix)]
```

Comment:
Windows has executable files too?

Reply:
It doesn’t seem like there’s a flag for executable files on Windows; my understanding is that contrarily to Linux where we need to set a file as executable, under Windows any normal file could be potentially executed, and the `.exe` (or `.msi` etc.) file extension is the only thing that can tell it’s supposed to be executed.

Reply:
Yea I agree with @ariasuni on this one. Detecting executables seems to be purely relying on file extensions, which is not elegant (orreliable)